rdrf/rdrf/templatetags/is_no_translation.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+from rdrf.models.definition.models import Registry
+
+register = template.Library()
+
+
+@register.filter
+def is_no_translation(registry):
```

Comment:
No longer a boolean return type so update the method name e.g. `get_translate_value`